api/conf/urls/__init__.py

```diff
@@ -9,6 +9,7 @@
 from django.contrib import admin
 from django.urls import include, path
 from django.views.generic import RedirectView
+from django.views.generic.base import TemplateView
```

Comment:
It's possible to combine the two imports.
```suggestion
from django.views.generic import RedirectView, TemplateView
```